ddtrace/vendor/wrapt/wrappers.py

```diff
@@ -822,6 +821,10 @@ def apply_patch(parent, attribute, replacement):
         setattr(parent, attribute, replacement)
     except (TypeError, AttributeError):
         # It is a built-in/extension type
+        # CAVEAT: Global import raises an error, i.e, asynctest package raises:
+        # 'NoneType' object has no attribute '_spec_coroutines'
```

Comment:
I think if would be good to have a link to the issue just so we can know if we have a regression in the future.

Reply:
PS: forget it, I saw you linked it in the changelog.

Reply:
Do we know what the issue is with `forbiddenfruit`? Is there an upstream issue we should be tracking?